src/sagemaker/image_uris.py

```diff
@@ -266,6 +269,7 @@ def _get_instance_type_family(instance_type):
 def _get_image_tag(
     container_version,
     distribution,
+    final_image_scope,
```

Comment:
Could you also update the docs in line 79? I'll ping the SDK team on Slack about updating the public docs.